MSTuringEmbeddingOnlyScenario.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+namespace VectorIndexScenarioSuite
+{ 
+    internal class MSTuringEmbeddingOnlyScenario : EmbeddingOnlyScearioBase

```

Comment:
Let us add an entry for MS MARCO web search too just to get it in. Its only a few more declarations like this, right?